docs/fields.md

```diff
@@ -146,7 +146,7 @@ Workflow is the definition of a workflow resource
 
 - [`memoize.yaml`](../examples/memoize.yaml)
 
-- [`nested-workflow.yaml`](../examples/nested-workflow.yaml)
+- [`nested-workflow.yaml`](https://github.com/argoproj/argo/blob/master/examples/examples/nested-workflow.yaml)
```

Comment:
example/examples???

Reply:
I have an example in `examples/memoize.yaml`, should I add more? 

I ran `codegen` again to fix the URL issue